src/xrpld/app/tx/detail/Batch.cpp

```diff
@@ -61,7 +58,7 @@ Batch::calculateBaseFee(ReadView const& view, STTx const& tx)
 
     // LCOV_EXCL_START
     if (baseFee > maxAmount - view.fees().base)
-        throw std::overflow_error("XRPAmount overflow");
+        return INITIAL_XRP;
```

Comment:
If it's related to a bug fix and it's covered by unit tests, can we remove LCOV_EXCL_START around it?

Reply:
If it is tested and covered yes we can remove the `LCOV_EXCL_START` but I believe none of these are tested except nested ttBATCH.